### PR TITLE
feat: record group insert at bottom when created

### LINKED ISSUE #9050: Record group adding a new row should insert at the bottom
Record group adding a new row should insert the record at the bottom, of the visible page.

### PR DESCRIPTION
**Pull request by magrinj**
Fix #9050 

When we add a new record in a record group this one should be added at them bottom.

### CODE REVIEW

File: packages/twenty-server/src/engine/api/graphql/graphql-query-runner/resolvers/graphql-query-create-one-resolver.service.ts [hunk 1]
```diff
@@ -83,8 +83,6 @@ export class GraphqlQueryCreateOneResolverService extends GraphqlQueryBaseResolv
     });
   }
 
-  async;
```
Comment (charlesBochet) [inline packages/twenty-server/src/engine/api/graphql/graphql-query-runner/resolvers/graphql-query-create-one-resolver.service.ts:86]: remove!

**Comment by charlesBochet** [comment]
@magrinj tested your PR, looks better and better!

Found a bug: if I have many pages, I add a record and then click on load more, my record is present twice
image

File: packages/twenty-front/src/modules/apollo/optimistic-effect/utils/triggerCreateRecordsOptimisticEffect.ts [hunk 2]
```diff
@@ -116,11 +128,61 @@ export const triggerCreateRecordsOptimisticEffect = ({
               );
 
               if (recordToCreateReference && !recordAlreadyInCache) {
-                nextRootQueryCachedRecordEdges.unshift({
+                const cursor = Buffer.from(
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply packages/twenty-front/src/modules/apollo/optimistic-effect/utils/triggerCreateRecordsOptimisticEffect.ts:131]: +add tests on it

**Review by charlesBochet — APPROVED** [review]

File: packages/twenty-front/src/modules/apollo/optimistic-effect/utils/triggerCreateRecordsOptimisticEffect.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (magrinj) [reply packages/twenty-front/src/modules/apollo/optimistic-effect/utils/triggerCreateRecordsOptimisticEffect.ts:131]: https://github.com/twentyhq/twenty/pull/9086

**Pull request merged**